<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_Calibration</class>
 <widget class="QWidget" name="Camera_Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_GetPic">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>标定图像</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pushButton_PicClear">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PicDir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像路径:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Browse">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trigger">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像采集</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PicDir">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>E:/Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_DealRestult">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>相机内参</string>
   </property>
   <widget class="QPushButton" name="pushButton_CalibrateInternal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标定内参</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DealRestult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标定结果:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DealRestult">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>E:/Image/Camera_Internal_Parameters</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BroRestult">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BroCalibrateBoard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibrateBoard">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>3.75caltab.descr</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_CalibrateBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标定板:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CalibrateError">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RMSE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibrateError">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>相机外参</string>
   </property>
   <widget class="QPushButton" name="pushButton_CalibrateExternal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标定外参</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CamInternalParamter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>内参:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CamInternalParamter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BroInternalParamter">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
